dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/ExportSecretForm.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.rest.api.v1.apps;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Set;
+import java.util.Map;
+
+public class ExportSecretForm extends Validated {
+
+    @NotNull
+    @Min(14)
+    @Max(32)
+    private final String password;
+
+    @NotNull
+    private final Map<String, Set<String>> appKeysBySite;
+
+    @NotNull
+    private final boolean exportAll;
+
+    @JsonCreator
+    public ExportSecretForm(@JsonProperty("password") final String password, @JsonProperty("exportAll") final boolean exportAll, @JsonProperty("appKeys") final Map<String, Set<String>> appKeysBySite) {
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5931733)